game/src/sandbox/gameplay/freeform/spawner.rs

```diff
@@ -49,8 +49,8 @@ impl AgentSpawner {
                     ),
                 ]),
                 Widget::row(vec![
-                    "Number of trips:".text_widget(ctx),
-                    Spinner::widget(ctx, "number", (1, 1000), 1),
+                    "Number of trips:".text_widget(ctx).centered_vert(),
```

Comment:
Thank you for your service!

I sometimes wonder if we should make the default text widget have a line height identical to our buttons so that we don't have to sprinkle around logic like this all the time.

It's a pretty common occurrence that our text labels don't line up with our controls -  e.g. the "Type of trip" dropdown above this control is still misaligned...

<img width="227" alt="Screen Shot 2021-04-29 at 10 49 10 AM" src="https://user-images.githubusercontent.com/217057/116595790-8902da80-a8d8-11eb-82fe-17e6f67926ff.png">

Reply:
I've thought about making `Widget::row` add `centered_vert` to all members that aren't specifying something else, but that's way more of a hack. I'll look into the text height default instead; that would be much nicer. And the widgetry demo has that place to test it out already!